### ISSUE TITLE
Please Update TRAINS_HOST_IP -> CLEARML_HOST_IP

### ISSUE THREAD

**Issue by michaelfuckner** [allegroai/clearml-server#68]
https://github.com/allegroai/clearml-server/blob/daade0894067575c65fcb0df0669f43488e5f77d/docker/docker-compose.yml#L125

**Comment by jkhenning** [comment]
Hi @michaelfuckner,
We're updating and testing it as we speak - I expect an update in a day or so 🙂

**Comment by michaelfuckner** [comment]
So I just wait some days before submitting patches?

**Comment by jkhenning** [comment]
Yes. This will be updated soon, and is only related to the Agent-Services service (i.e. required a rebuild of the agent-services docker images, which we're doing 🙂)
